backend/src/passwords.cpp

```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include "passwords.hpp"
+
+namespace passwords
+{
+    std::unordered_set<std::string> generatePasswords(const int &numPasswords, const int &maxChars)
+    {
+        int minChars = 10;
+
+        if (maxChars < minChars) {
+            std::cout << "Password must have a minimum length of 10 characters\n";
+            exit(0);
+        }
```

Comment:
**issue**: This is the C-style way of handling exceptions. C++ adds `throw` and `try` `catch` blocks. Please use that instead.